update to doc to make copy and past easier and to add more info (#564)
Kuadrant/multicluster-gateway-controller: docs/gateways/define-and-place-a-gateway.md

In this guide, we will go through defining a Gateway in the OCM hub cluster that can then be distributed to and instantiated on a set of managed spoke clusters.

### Prerequisites
* Complete the [Getting Started Guide](https://docs.kuadrant.io/getting-started/) to bring up a suitable environment. 

If you are looking to change provider from the default Istio:
* Please have the Gateway provider of your choice installed and configured (in this example we use Envoy gateway. See the following [docs](https://gateway.envoyproxy.io/v0.5.0/user/quickstart.html))

> ## Inital setup 

Reviewer: ```suggestion
## Initial setup 
```

    ```

    You'll see the following:

    ```
    kuadrant-multi-cluster-gateways   prod-web   istio                                         172.31.200.0                29s
    multi-cluster-gateways            prod-web   kuadrant-multi-cluster-gateway-instance-per-cluster                  True         2m42s
    ```
3.  Execute the following to see the gateway on the workload-1 cluster:

    ```bash
    kubectl --context kind-mgc-workload-1 get gateways -A
    ```
    You'll see the following
    ```
    NAMESPACE                         NAME       CLASS   ADDRESS        PROGRAMMED   AGE
    kuadrant-multi-cluster-gateways   prod-web   istio   172.31.201.0                90s
    ```
### Using a different gateway provider?

While we recommend using Istio as the gateway provider as that is how you will get access to the full suite of policy APIs, it is possible to use another provider if you choose to however this will result in a reduced set of applicable policy objects.

If you are only using the DNSPolicy and TLSPolicy resources, you can use these APIs with any Gateway provider. To change the underlying provider, you need to set the gatewayclass param `downstreamClass`. 

> 1.  Create the following configmap (eg stands for envoy gateway see above prerequisites):

Reviewer: ```suggestion
1.  Create the following configmap. Note: In this example, 'eg' stands for the Envoy gateway, which is mentioned in the prerequisites above:
```

In this guide, we will go through defining a Gateway in the OCM hub cluster that can then be distributed to and instantiated on a set of managed spoke clusters.

### Pre Requisites
* Complete the [Getting Started Guide](https://docs.kuadrant.io/getting-started/) to bring up a suitable environment. Going through the installation will ensure that a supported `GatewayClass` is registered in the hub cluster that the Kuadrant multi-cluster gateway controller will handle. 

If you are looking to change provider from the default Istio:

> * Please have the Gateway provider of your choice installed and configured (in this example we use Envoy gateway see the following [docs](https://gateway.envoyproxy.io/v0.5.0/user/quickstart.html))

Reviewer 2: ```suggestion
* Please have the Gateway provider of your choice installed and configured (in this example we use Envoy gateway. See the following [docs](https://gateway.envoyproxy.io/v0.5.0/user/quickstart.html))
```

# Open terminal sessions and set cluster context

For this walkthrough, we're going to use multiple terminal sessions/windows.

Open two windows, which we'll refer to throughout this walkthrough as:

* `T1` (Hub Cluster)
* `T2` (Workloads cluster)

Set the kubecontext for each terminal, refer back to these commands if re-config is needed.

In `T1` run `kind export kubeconfig --name=mgc-control-plane --kubeconfig=$(pwd)/control-plane.yaml && export KUBECONFIG=$(pwd)/control-plane.yaml`

In `T2` run `kind export kubeconfig --name=mgc-workload-1 --kubeconfig=$(pwd)/workload1.yaml && export KUBECONFIG=$(pwd)/workload1.yaml`

export `MGC_SUB_DOMAIN` in each terminal if you haven't already added it to your `.zshrc` or `.bash_profile`.

> **NOTE** :exclamation: The quick start script will create a placement resource as part of the setup you can use this as further inspiration for other placement resources you would like to create.

Reviewer 2: ```suggestion
**NOTE** :exclamation: The quick start script will create a placement resource as part of the setup. You can use this as further inspiration for other placement resources you would like to create.
```
